charts/osm/templates/osm-deployment.yaml

```diff
@@ -38,14 +38,9 @@ spec:
       initContainers:
         - name: init-osm-controller
           image: {{ .Values.osm.curlImage }}
+          command: ["sh", "-c"]
           args:
-          - /bin/sh
-          - -c
-          - >
-            set -x;
-            while [ $(curl --connect-timeout 2 -sw '%{http_code}' "http://osm-bootstrap.{{ include "osm.namespace" . }}.svc.cluster.local:9095/healthz" -o /dev/null) -ne 200 ]; do
-              sleep 10;
-            done
+            - curl --connect-timeout 2 "http://osm-bootstrap.{{ include "osm.namespace" . }}.svc.cluster.local:9095/healthz" --retry 50 --retry-connrefused --retry-delay 5
```

Comment:
If the idea here is to remove the dependency on `sh` in the image, we could call `curl` directly in the `command`.

Reply:
+1

Let's avoid using `sh` as the entrypoint for the command.

Reply:
thanks for the suggestion @nojnhuh and @shashankram, I have updated the code. PTAL